bin/cli.ml

```diff
@@ -12,7 +12,7 @@ let non_deterministic =
 
 let syntax =
   let parse = function
-    | "normal" -> `Ok Mdx.Normal
+    | "markdown" | "normal" -> `Ok Mdx.Normal
     | "cram" -> `Ok Mdx.Cram
```

Comment:
What do you think of moving this to `Syntax.of_string : string -> t option` ?
Because you improved and exposed that module, I think it makes sense to also expose that code.

Reply:
It's a good idea, done.